tests/test_engines.py

```diff
@@ -73,8 +73,8 @@ def test_espeak_voices(driver_name):
     no_nyc = expected.rpartition("|")[0]
     assert name_str in {expected, no_nyc}, f"Expected '{expected}' or '{no_nyc}'."
     print(f"({name_str.replace('|', ' ; ')})", end=" ", flush=True)
-    engine.runAndWait()
-    engine.setProperty("voice", voice)  # Reset voice to original value
```

Comment:
Listen to the tests that come after this one to see why restoring defaults makes things more intuitive.